_template_python/INDICATOR_DEV_GUIDE.md

```diff
@@ -0,0 +1,446 @@
+
+[Example analysis 1](https://github.com/cmu-delphi/covidcast-indicators/pull/1495#issuecomment-1039477646), [example analysis 2](https://github.com/cmu-delphi/covidcast-indicators/issues/367#issuecomment-717415555).
+
+Once the analysis is complete, have the stakeholder (usually the original requestor of the data source, can also be [@RoniRos](https://www.github.com/RoniRos)) review it.
+
+### Documentation
+
+The [documentation site](https://cmu-delphi.github.io/delphi-epidata/) ([code here](https://github.com/cmu-delphi/delphi-epidata/tree/628e9655144934f3903c133b6713df4d4fcc613e/docs)) stores long-term long-form documentation pages for each indicator, including those that are inactive.
+
+Active and new indicators go in the [COVIDcast Main Endpoint -> Data Sources and Signals](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_signals.html) section ([code here](https://github.com/cmu-delphi/delphi-epidata/tree/628e9655144934f3903c133b6713df4d4fcc613e/docs/api/covidcast-signals)).
+A [template doc page](https://github.com/cmu-delphi/delphi-epidata/blob/628e9655144934f3903c133b6713df4d4fcc613e/docs/api/covidcast-signals/_source-template.md) is available in the same directory.
+
+An indicator documentation page should contain as much detail (including technical detail) as possible.
+The following fields are required:
+
+* Description of the data source and data collection methods
+* Links to the data source (organization and specific dataset(s) used)
+* Links to any data source documentation you referenced
+* List of signal names, descriptions, with start dates
+* Prose description of how signals are calculated
+* Specific math showing how signals are calculated, if unusual or complex or you like equations
+* How smoothing is done, if any
+* Known limitations of the data source and the final signals
+* Missingness characteristics, especially if the data is missing with a pattern (on weekends, etc)
+* Lag and revision characteristics
+* Licensing information
+
+and anything else that changes how users would use or interpret the data, impacts the usability of the signal, may be difficult to discover, recommended usecases, is unusual, any gotchas about the data or the data processing approach, etc.
+_More detail is better!_
+
+At the time that you're writing the documentation, you are the expert on the data source and the indicator.
+Making the documentation thorough and clear will make the data maximally usable for future users, and will make maintenance for Delphi easier.
+
+(For similar reasons, comment your code extensively!)
+
+## Step 3: Deployment
+
+* This is after we have a working one-off script 
+* Using Delphi utils and functionality 
+* What happens to the data after it gets put in `receiving/`:
+
+Next, the `acquisition.covidcast` component of the `delphi-epidata` codebase does the following immediately after an indicator run (you need to set acquisition job up):
```

Comment:
> (you need to set acquisition job up)

Do we have docs on how to do this at all? I'm still hazy on it. I guess it's the link down in ### Staging

Reply:
Minh [added deployment info](https://github.com/cmu-delphi/covidcast-indicators/pull/1986) that should cover that.